README.md

```diff
@@ -2,55 +2,158 @@
 
 ![Dolphin Guy](http://i.giphy.com/RdkwfjGlsdnRm.gif)
 
-## Learning Objectives - The student should be able to..
+## Objectives
 
-* Explain that the `Bool` is a data type where the value can only be **true** or **false**.
-* Create variables using the Boolean literal values, `true` or `false` like so:
+1. Describe the `Bool` data type
+1. Create Boolean variables with literal values
+1. Use comparison operators on Boolean variables
+1. Use logic operators on Boolean variables
+
+## Understanding Booleans
+
+You are already familiar with creating `Int` variables and constants, which can store single numeric integer values, like
+
+```swift
+var numberOfApples = 7
+let numberOfOranges = 12
+```
+
+In this lesson, we will learn about the boolean data type, which represents a value that can either be true or false. Swift provides constant literal values for both possibilities, which can be used like the integer literals you are already familiar with:
+
+```swift
+let isTheSkyBlue = true
+let canHumansFly = false
+```
+
+Creating and using a variable of this type also works like the types you already know:
+
+```swift
+var isTheGrassGreen = false
+isTheGrassGreen = true
+```
+
+In both of these examples, the Swift compiler has inferred the type from our use of the literal values, but we can also explicitly write out the type if we want to:
+
+```swift
+let canDolphinsWalk: Bool = false
+```
+
+## Comparing values
+
+A common use of the `Bool` type is as a result of a comparison of other values. Comparing two values in Swift is done with an *operator*, which is a special symbol that you use to check, combine or change one or more values.
+
+For example, we can compare the two integer variables from our earlier example, to see if they are equal, using the equal-to operator:
+
+```swift
+let doWeHaveAsManyApplesAsOranges = numberOfApples == numberOfOranges // false, because 7 is not equal to 12
+```
+
+The result in this case is of course `false`, because 7 and 12 are not equal numbers. We can also determine if two values are **not** equal, like this:
+
+```swift
+let compareApplesToOranges = numberOfApples != numberOfOranges // true, because 7 is indeed not equal to 12
+```
+
+In this case, the result is `true`. We may also be interested in learning if we have more apples than oranges, for this we have operators for greater and less than:
+
+```swift
+let moreApplesThanOranges = numberOfApples > numberOfOranges // false, because 7 is not greater than 12
+let lessApplesThanOranges = numberOfApples < numberOfOranges // true, because 7 is less than 12
+```
+
```

Comment:
I like how you started this off posing the question, "We may also be interested in learning if we have more apples than oranges.."

Instead though of storing the evaluation to a variable, I think we can more concise as to not confuse a few who might get hung up on seeing the assignment operator so close to the `==` operator.
```swift
numberOfApples > numberOfOranges // false
// Is the numberOfApples (7) greater than the numberOfOranges (12)? The answer is no, which means this evaluates to false.

numberOfApples < numberOfOranges // true
// Is the numberOfApples (7) less than the numberOfOranges (12)? The answer is yes, which means this evaluates to true.
```